ronin/scripts/vscripts/ui/menu_past_runs.nut

```diff
@@ -66,6 +66,7 @@ void function InitPastRunsMenu()
                     if (!run.isPB && i >= 50 && run.goldSplits.len() <= 0)
                     {
                         DeleteRun(run)
+                        --i
```

Comment:
nvm, im stupid, remove this line and replace the i++ with an i-- at the beginning of the for loop